Sources/StreamChatUI/Generated/L10n.swift

```diff
@@ -14,139 +10,131 @@ import Foundation
 // swiftlint:disable explicit_type_interface function_parameter_count identifier_name line_length
 // swiftlint:disable nesting type_body_length type_name vertical_whitespace_opening_braces
 internal enum L10n {
```

Comment:
This re-generated by accident or not? Otherwise I don't understand this :(

Reply:
Nope, this is the version of the file without the formatter. It was committed formatted originally.

Reply:
From now on you should see changes in this file only if you actually change something there.

Reply:
Great! Thank you❗️